### PR TITLE
Update turn-howto.md

### PR DESCRIPTION
**Pull request by mlakkadshaw**
### Pull Request Checklist

* [x] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#changelog). The entry should:
  - Be a short description of your change which makes sense to users. "Fixed a bug that prevented receiving messages from other servers." instead of "Moved X method from `EventStore` to `EventWorkerStore`.".
  - Use markdown where necessary, mostly for `code blocks`.
  - End with either a period (.) or an exclamation mark (!).
  - Start with a capital letter.
  - Feel free to credit yourself, by adding a sentence "Contributed by @github_username." or "Contributed by [Your Name]." to the end of the entry.
* [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
* [x] [Code style](https://matrix-org.github.io/synapse/latest/code_style.html) is correct
  (run the [linters](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#run-the-linters))

### CODE REVIEW

**Comment by DMRobertson** [comment]
@mlakkadshaw did you mean to close this?

**Comment by mlakkadshaw** [comment]
I was having problem creating the changelog hence closed it. Will create changelog file and re-open.

File: changelog.d/12254.doc
Comment (H-Shay) [inline changelog.d/12254.doc:1]: Thanks for this! Looks like the CI is failing due to a lack of punctuation here. Also generally changelog entries take the imperative form:
```suggestion
Update turn docs with instructions on how to use a free turn server from open relay project with Synapse.
```

Comment (H-Shay) [inline changelog.d/12254.doc:1]: Thanks for this! Looks like the CI is failing due to a lack of punctuation here. Also generally changelog entries take the imperative form:
```suggestion
Update turn docs with instructions on how to use a free turn server from open relay project with Synapse.
```

Comment (mlakkadshaw) [reply changelog.d/12254.doc]: Thanks, I have updated the changelog entry.

File: docs/turn-howto.md
Comment (richvdh) [inline docs/turn-howto.md:256]:
```suggestion
Here is an example Synapse configuration:
```

File: docs/turn-howto.md
Comment (richvdh) [inline docs/turn-howto.md:265]: This line seems to say the same thing as above? or am I missing something?

Comment (richvdh) [reply docs/turn-howto.md:265]: oh, the port numbers are different. and they are missing the `turn:` scheme. I'm confused - which should I use?

File: docs/turn-howto.md
Comment (richvdh) [inline docs/turn-howto.md:267]: yes, but I don't think that's relevant here?

**Review by richvdh — CHANGES_REQUESTED** [review]
Thanks for this!

Generally it seems like a nice idea to mention this, but the problem is that it doesn't really fit into the document as it stands. For example, there is a paragraph in the introduction:

> The following sections describe how to install coturn ... and integrate it with synapse.

... and this new section is something completely different. Inconsistencies like this make it very hard for a reader to understand what they they are supposed to do: it makes it look like they need to do this extra stuff *as well as* setting up coturn.

An alternative approach might be to put this in a completely different document - "Using Synapse with Open Relay" - and then just including a link to it in `turn-howto.md`.

**Comment by richvdh** [comment]
Please also be aware that we require sign-off for any contributions, per https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off.

File: docs/turn-howto.md
Comment (mlakkadshaw) [reply docs/turn-howto.md:265]: Generally it is recommended to use both the uri one with port 80 and anot another with port 443.

I think there could be a scenario where port 80 is blocked by the firewall and only port 443 is allowed but not vice versa.

So if we want to keep just one then I would keep the uri with port 443.

sorry I forgot the `turn:` scheme

**Comment by richvdh** [comment]
> but where it would fit in the hierarchy of current documentation?

I'm afraid I don't understand this question. What do you mean by the "hierarchy"?

**Comment by clokep** [comment]
@mlakkadshaw Do you have interest in continuing this PR? It looks like previous review feedback was not handled.

**Comment by clokep** [comment]
Closing due to lack of response.

**Pull request closed without merging**
